### PR TITLE
add resize check

### PR DESCRIPTION
**Pull request by andrejikonikov**
**Description of changes:**
this branch will add resize handler for thumbnails component

### CODE REVIEW

File: src/components/PaginatedThumbnails.js [hunk 1]
```diff
@@ -13,6 +13,15 @@ export default class PaginatedThumbnails extends Component {
...
 			width: 500,
 		};
+
+		this.container = null;
+
+		this.setContainerRef = this.setContainerRef.bind(this);
+		this.updateWidth = this.updateWidth.bind(this);
+	}
+
+	setContainerRef (e) {
```
Comment (RokasC) [inline src/components/PaginatedThumbnails.js:23]: Why not use `React.createRef ` https://reactjs.org/docs/react-api.html#reactcreateref ?

Comment (andrejikonikov) [reply src/components/PaginatedThumbnails.js:23]: it is not a function, I think it is because of react version

Comment (bstst) [reply src/components/PaginatedThumbnails.js:23]: `createRef` should work here.
Create it in the constructor (`this.container = createRef()`), reference it in the component (`<div ref={this.container}`), and use it where you need to (`this.setState({ width: this.container.current.offsetWidth });`)

**Review by bstst — CHANGES_REQUESTED** [review]

**Comment by andrejikonikov** [comment]
@bstst @RokasC  I rather know how refs should be used, the thing is, I can't make it work in usual way. What I get instead is: `TypeError: _react2.default.createRef is not a function`. Any thoughts on this?

**Comment by RokasC** [comment]
React version is too old for that. We can live how it is done now I guess

**Review by RokasC — APPROVED** [review]

**Review by bstst — APPROVED** [review]

**Pull request merged**